src/_P052_SenseAir.ino

```diff
@@ -556,23 +526,23 @@ boolean Plugin_052(uint8_t function, struct EventStruct *event, String& string)
                        period);
                 success = true;
               }
-            } else if (equals(subcommand, F("setrelay"))) {
-              int state = 0;
-
-              if (validIntFromString(parseString(string, 3), state)) {
-                short relaystatus = 0; // 0x3FFF represents 100% output.
-                success = true;
-
-                //  Refer to sensor model’s specification for voltage at 100% output.
-                if (state == 0) { relaystatus = 0; }
-                else if (state == 1) { relaystatus = 0x3FFF; }
-                else if (state == 1) { relaystatus = 0x7FFF; }
-                else { success = false; }
-
-                if (success) {
-                  P052_data->modbus.writeSingleRegister(0x18, relaystatus);
-                  addLog(LOG_LEVEL_INFO, concat(F("Senseair command: relay="), state));
-                }
+            }
+          } else if (equals(subcommand, F("setrelay"))) {
+            int state = 0;
+
+            if (validIntFromString(parseString(string, 3), state)) {
+              short relaystatus = 0; // 0x3FFF represents 100% output.
+              success = true;
+
+              //  Refer to sensor model’s specification for voltage at 100% output.
+              if (state == 0) { relaystatus = 0; }
+              else if (state == 1) { relaystatus = 0x3FFF; }
+              else if (state == 1) { relaystatus = 0x7FFF; }
+              else { success = false; }

```

Comment:
That looks a bit strange, checking for `state == 1` twice?

Reply:
No, should be -1 ;)